src/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper/BootstrapperHostingStartup.cs

```diff
@@ -35,7 +35,7 @@ internal class BootstrapperHostingStartup : IHostingStartup
     /// </summary>
     public BootstrapperHostingStartup()
     {
-        _settings = Instrumentation.LogSettings;
+        _settings = Instrumentation.LogSettings.Value;
```

Comment:
not from your change but this private should be named `_logSettings`.

Reply:
fixed 0d65ba67f55f8e1d57d1d1d68d7856fe006e11b0